nbs/core.ipynb

```diff
@@ -292,7 +292,6 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #583.            for i_window, (cutoffs, train, test) in enumerate(splits):
</pre><p> </p><p>This is a comment for future us. I foresee that pre-computing and keeping in memory all splits can cause memory issues if datasets are large and it has many splits, since dataset is almost multiplied number of splits times. Is there any advantage on this approach vs creating and keeping in memory only the current split? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/842/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
The backtest_splits [returns a generator](https://github.com/Nixtla/utilsforecast/blob/main/utilsforecast/processing.py#L740), so this won't be an issue.